gwcli/client.py

```diff
@@ -475,10 +475,6 @@ def ui_command_disk(self, action='add', disk=None, size=None):
             if action == 'add':
                 ui_root = self.get_ui_root()
                 ui_disks = ui_root.disks
-                if not size:
```

Comment:
Can the ```size``` parameter (from the CLI action) be dropped now?

Reply:
The `size` parameter is still needed if the RBD image does not exist yet, and we want to create it using `ceph-iscsi` (this PR is just changing the parameter from mandatory to optional).

Dropping this parameter would require the drop of the functionality that creates RBD images directly from the `ceph-iscsi`.

Reply:
OK, I guess I didn't realize that you could create brand new disks from under the "host/XYZ" UI tree